pulp_container/app/webserver_snippets/apache.conf

```diff
@@ -1,5 +1,8 @@
-ProxyPass /v2 http://${pulp-content}/v2
-ProxyPassReverse /v2 http://${pulp-content}/v2
+ProxyPass /v2 http://${pulp-api}/v2
+ProxyPassReverse /v2 http://${pulp-api}/v2
```

Comment:
Does this mean, we are moving the whole live api from the content-app to the api-app now?

Reply:
Yes, the REST API for the container registry is going to be served by pulpcore-api app. pulpcore-content is only going to serve actual content and not the REST API.